protos/block_and_transaction.proto

```diff
@@ -72,13 +72,13 @@ message UserAdministrationTxEnvelope {
 
 
 message DataTx {
-  repeated string mustSignUserIDs = 1;
-  string txID = 2;
-  repeated DBOperation DBOperations = 3;
+  repeated string must_sign_user_ids = 1;
+  string tx_id = 2;
+  repeated DBOperation db_operations = 3;
 }
 
 message DBOperation {
```

Comment:
Just one question - shouldn't we change `DBOperation`, `DBAdministrationTx`, `DBIndex`, `GetDBStatusQuery` change to same way as we changed fields names?

Reply:
No, we don't need to change the message name. Only the field names are used when we do `json.Marshal()`